Tabby.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = "Tabby"
-  s.summary          = "If cats looked like frogs we'd realize what nasty, cruel little bastards they are. Style. That's what people remember."
```

Comment:
You don't like this description, don't you?

Reply:
😄